server/views/macros/activateSuspendedPunishmentsTable.njk

```diff
@@ -16,7 +16,7 @@
             classes: "columnWidth30 verticalAlign"
           },
           {
-            text: punishment.reportNumber,
+            text: punishment.chargeNumber,
```

Comment:
this is actually a bug which must be in live from when we switched from report number to charge number!